lib/gateway.js

```diff
@@ -140,6 +148,18 @@ async function buildGateway (gatewayOpts) {
         typeFieldsToService[`${type}-${field}`] = service
       }
     }
+
+    factory.add(`${service}Entity`, async (queries, context) => {
+      return queries.map(({ query, variables }) => {
```

Comment:
This should be one single http request with the same entity, or possibly coalescing all the queries into one.

Reply:
Done in the latest commit